enhancements/console/customize-perspectives.md

```diff
@@ -0,0 +1,278 @@
+---
+title: customize-perspectives
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@jerolimov"
+  - "@cvogt"
+creation-date: 2022-07-26
+last-updated: 2022-08-24
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Perspectives 
+
+## Summary
+
+This document provides details about how cluster administrator 
+can hide user perspectives. 
+
+## Motivation
+
+Cluster administrator wants to hide the "Administrator" perspective 
+from non-privileged or all users and to hide the "Developer" 
+perspective for all users.
+
+### User Stories
+
+#### Story 1
```

Comment:
A user story typically looks like `<someone> wants to be able to <something>, so that they can <something>`, think of it as a who, what and why. Currently, you're missing the `so that they can <something>` from your stories. Please tell me why this is important and why the user would want to these things